e2e/cypress/integration/channel_sidebar/dm_gm_behaviour_spec.js

```diff
@@ -0,0 +1,257 @@
+                cy.get('#headerTeamName').should('be.visible');
+
+                // # Post a message as the new user
+                cy.postMessageAs({
+                    sender: user,
+                    message: `Hey ${testUser.username}`,
+                    channelId: channel.id,
+                });
+
+                // * Verify that a DM channel shows up in the sidebar
+                cy.get(`.SidebarLink:contains(${user.username})`).should('be.visible').click();
+
+                // # Close the new DM
+                cy.get('#channelHeaderDropdownButton').click();
+                cy.get('#channelCloseMessage').click();
+
+                // * Verify that the DM channel disappears
+                cy.get(`.SidebarLink:contains(${user.username})`).should('not.be.visible');
+
+                // # Post a message as the new user again
+                cy.postMessageAs({
+                    sender: user,
+                    message: `Hello ${testUser.username}`,
+                    channelId: channel.id,
+                });
+
+                // * Verify that the DM channel re-appears in the sidebar
+                cy.get(`.SidebarLink:contains(${user.username})`).should('be.visible');
+            });
+        });
+    });
+
+    it('MM-T2017_1 Opening a new GM should make sure the GM appears in the sidebar', () => {
+        // # Create 2 new users to start a GM with
+        cy.apiCreateUser().then(({user}) => {
+            cy.apiCreateUser().then(({user: user2}) => {
+                // * Verify that we can see the sidebar
+                cy.get('#headerTeamName').should('be.visible');
+
+                // # Click the + button next to the DM category
+                cy.get('.SidebarChannelGroupHeader_addButton').should('be.visible').click();
+
+                // # Search for the new user's username
+                cy.get('#selectItems input').
+                    type(user.username, {force: true}).
+                    wait(TIMEOUTS.HALF_SEC);
+
+                // # Select the user you searched for
+                cy.get(`#displayedUserName${user.username}`).click().wait(TIMEOUTS.HALF_SEC);
+
+                // # Search for the 2nd user's username
+                cy.get('#selectItems input').
+                    type(user2.username, {force: true}).
+                    wait(TIMEOUTS.HALF_SEC);
+
+                // # Select the user you searched for
+                cy.get(`#displayedUserName${user2.username}`).click().wait(TIMEOUTS.HALF_SEC);
+
+                // # Click Go
+                cy.findByTestId('saveSetting').should('be.visible').click();
+
+                // * Verify that a GM channel shows up in the sidebar
+                cy.get(`.SidebarLink:contains(${user.username})`).should('contain', user2.username).should('be.visible');
+            });
+        });
+    });
+
+    it('MM-T2017_2_3 Receiving a DM from a user should show the DM in the sidebar', () => {
```

Comment:
Same here. Also this case is related to GM. Should it use GM instead of DM?
```suggestion
    it('MM-T2017_2 Receiving a DM from a user should show the DM in the sidebar', () => {
```